src/frontend/shortlist/src/App.vue

```diff
@@ -119,7 +147,7 @@ function appAddStudent(payload) {
   <NavBar />
   <div class="app-container">
     <RouterView
-      @appAccountLogin="appAccountLogin"
+      @appAccountLogin="appAuthLogin"
       @appAccountSignup="appAccountSignup"
```

Comment:
From Account to Auth, does this mean this pr has moved things for login/signup from one backend to the other completely?

Reply:
For login/signup, yes. This calls the API at /auth/login instead of /account/login